pkg/cli/cliflags/flags.go

```diff
@@ -160,7 +160,7 @@ accept requests.`,
 Maximum memory capacity available to store temporary data for SQL clients,
 including prepared queries and intermediate data rows during query execution.
 Accepts numbers interpreted as bytes, size suffixes (e.g. 1GB and 1GiB) or a
-percentage of physical memory (e.g. 25%).
+percentage of physical memory (e.g. 25% or .25).
```

Comment:
There's no need to give two examples; I would just use the decimal form here (and s/percentage/fraction/). We can give both forms in the full documentation but other than backwards compatibility there's no reason for someone to use the percentage form instead of the decimal.

Reply:
Done.